ionoscloud/resource_vcpu_server_test.go

```diff
@@ -109,6 +110,7 @@ func TestAccServerVCPUBasic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckServerVCPUExists(constant.ServerVCPUResource+"."+constant.ServerTestResource, &server),
 					resource.TestCheckResourceAttr(constant.ServerVCPUResource+"."+constant.ServerTestResource, "name", constant.ServerTestResource),
+					resource.TestCheckResourceAttr(constant.ServerVCPUResource+"."+constant.ServerTestResource, "hostname", constant.ServerTestHostname),
```

Comment:
@cristiGuranIonos same, maybe add an update test for the `hostname` value.

Reply:
added